catalogs/catalog-lakehouse-paimon/src/test/java/org/apache/gravitino/catalog/lakehouse/paimon/TestGravitinoPaimonTable.java

```diff
@@ -201,6 +204,80 @@ void testCreatePaimonTable() {
             .contains(String.format("Paimon table %s already exists", tableIdentifier)));
   }
 
+  @Test
+  void testCreatePaimonPartitionedTable() {
+    String paimonTableName = "test_paimon_partitioned_table";
+    NameIdentifier tableIdentifier = NameIdentifier.of(paimonSchema.name(), paimonTableName);
+    Map<String, String> properties = Maps.newHashMap();
+    properties.put("key1", "val1");
+    properties.put("key2", "val2");
+
+    GravitinoPaimonColumn col1 =
+        fromPaimonColumn(new DataField(0, "col_1", DataTypes.INT().nullable(), PAIMON_COMMENT));
+    GravitinoPaimonColumn col2 =
+        fromPaimonColumn(new DataField(1, "col_2", DataTypes.DATE().notNull(), PAIMON_COMMENT));
+    RowType rowTypeInside =
+        RowType.builder()
+            .field("integer_field_inside", DataTypes.INT().notNull())
+            .field("string_field_inside", DataTypes.STRING().notNull())
+            .build();
+    RowType rowType =
```

Comment:
could we remove `rowType` in this test? this will the test focus on the partition related test and make it more clear.

Reply:
fixed.